requirements.txt

```diff
@@ -1,10 +1,8 @@
-click==7.1.2
+docopt
 Flask==1.1.2
-itsdangerous==1.1.0
 Jinja2==2.11.2
 MarkupSafe==1.1.1
 pyyaml==5.3.1
 requests==2.25.1
 ruamel.yaml==0.16.12
-ruamel.yaml.clib==0.2.2
-Werkzeug==1.0.1
\ No newline at end of file
+ruamel.yaml.clib==0.2.2
```

Comment:
Curious how you updated this. I know you typically do `pip freeze > requirements.txt`, but I believe that includes everything installed in your venv, which might include stuff that's not used in the code.